src/Core/src/Primitives/LayoutAlignment.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Microsoft.Maui.Primitives
+{
+	// We don't use Microsoft.Maui.Controls.LayoutAlignment directly because it has a Flags attribute, which we do not want
```

Comment:
Is it even possible to have multiple of these? What is `Start | End` even doing?

Actually, the enum in the Controls is invalid. It is _not_ a flaggable enum:
 - `Start | Center` == `Center`
 - `Start | End` == `End`
 - `Center | End` == `Fill`

We might be better off just removing the `[Flags]` and at that point it is almost the same, just with different ordering.

Reply:
Yes, the Controls one is weird/incorrect. But removing the FlagsAttribute would be a breaking change, and it's not on the breaking changes list. We can always obsolete the Controls version in a later release and sub in the Core version. Or we can open a separate discussion about adding it as a breaking change.

>  just with different ordering. 

This is the other reason I wanted a new enum. The enum in Controls has a default value of `Start`, but View defaults both values to `Fill` (WPF does this too). I wanted the framework default to match the enum default.